pkgs/games/vdrift/default.nix

```diff
@@ -33,10 +34,22 @@ let
       sha256 = "001wq3c4n9wzxqfpq40b1jcl16sxbqv2zbkpy9rq2wf9h417q6hg";
     };
 
-    nativeBuildInputs = [ pkg-config sconsPackages.scons_3_1_2 ];
+    nativeBuildInputs = [ pkg-config sconsPackages.scons_latest ];
     buildInputs = [ libGLU libGL SDL2 SDL2_image libvorbis bullet curl gettext ];
 
-    patches = [ ./0001-Ignore-missing-data-for-installation.patch ];
+    patches = [
+      ./0001-Ignore-missing-data-for-installation.patch
+      (fetchpatch {
+        name = "scons-python-3-fixes.patch";
+        url = "https://github.com/VDrift/vdrift/commit/2f1f72f2a7ce992b0aad30dc55509b966d1bb63d.patch";
+        sha256 = "sha256-gpIB95b1s+wpThbNMFXyftBPXkZs9SIjuCcvt068uR8=";
+      })
+      (fetchpatch {
+        name = "sconstruct-python-3-fix.patch";
+        url = "https://github.com/VDrift/vdrift/commit/7d04c723a165109e015204642f4984a1a4452ccb.patch";
+        sha256 = "sha256-ASEV46HnR90HXqI9SgHmkH2bPy5Y+vWN57vEN4hJMts=";
+      })
```

Comment:
Shouldn't we just bump it to the latest version instead?

Reply:
The latest version crashes on my machine, so I prefer to leave the work to the maintainer.